test_env_vars.yaml

```diff
@@ -43,3 +43,15 @@ dep_replacement:
   - name: github.com/Masterminds/semver
     type: gomod
     version: v1.4.0
+# Test data for the Using cached dependencies test
+cached_dependenices:
+  # The URL for the upstream git repository
+  seed_repo: "https://github.com/release-engineering/retrodep.git"
+  # Package that will be used for the test
+  package:
```

Comment:
Grouping `repo` and `pkg_managers` under `package` doesn't sound right to me. How about something like:
```yaml
test_repo:
  ssh_url: git@github.com:user/cachito_test_repo.git
  https_url: https://github.com/user/cachito_test_repo.git
  pkg_managers: ["gomod"]
```

For consistency, you could use the same structure for the `seed_repo`:
```yaml
seed_repo:
  https_url: https://github.com/release-engineering/retrodep.git
```